Expensify/App
# [No QA] Fix Flipper for Android

### Details
Flipper was not working on Android

### Fixed Issues
No Issue

### Tests
1. Build Android app on dev (use actual device if possible)
2. Verify Flipper works.

[**Huh? What is Flipper?**](https://stackoverflow.com/c/expensify/questions/10490)

### QA Steps
None

### Tested On


### Screenshots
❌
